sympy/physics/control/lti.py

```diff
@@ -426,7 +342,8 @@ def is_biproper(self):
         return degree(self.num, self.var) == degree(self.den, self.var)
 
 
-class Series(Basic):
+class Series(TransferFunctionExpr):
```

Comment:
Does subclassing from TransferFunctionExpr now solve the problems we were having with  `tf1 + tf2 - tf3 + tf4`?